lib/migrations/add_cross_assignments_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddCrossAssignmentsTable < ActiveRecord::Migration
+  def change
+    create_table :cross_assignments do |t|
+      t.integer :cross_parent_id, null: false
```

Comment:
Just to document our offline conversation, I originally had it that way, but I changed it because the AR PE's relations rely (or at least benefit greatly from) using the exact column names. Keeping the names different across tables allows us to avoid a naming conflict here so that calling `.parents` on a PE will always be clear as to what's returned.